lua/orgmode/export/init.lua

```diff
@@ -111,11 +111,15 @@ function Export.emacs(format, extension)
   if vim.fn.executable('emacs') ~= 1 then
     return utils.echo_error('emacs executable not found. Make sure emacs is in $PATH.')
   end
+  local emacs = config.emacs_config.executable_path
```

Comment:
Move this above to line 110, and include it in the check on line 111 instead of `'emacs'` string.

Reply:
Done